pkgs/top-level/all-packages.nix

```diff
@@ -11089,9 +11089,15 @@ with pkgs;
 
   perceptualdiff = callPackage ../tools/graphics/perceptualdiff { };
 
-  inherit (import ../servers/sql/percona-server pkgs) percona-server_lts percona-server_innovation;
+  inherit (import ../servers/sql/percona-server pkgs) percona-server_8_0 percona-server_8_4;
+  percona-server_lts = percona-server_8_4;
+  # temporarily, latest LTS and Innovation release are equal
+  percona-server_innovation = percona-server_8_4;
   percona-server = percona-server_lts;
-  inherit (import ../tools/backup/percona-xtrabackup pkgs) percona-xtrabackup_lts percona-xtrabackup_innovation;
+  inherit (import ../tools/backup/percona-xtrabackup pkgs) percona-xtrabackup_8_0 percona-xtrabackup_8_4;
+  percona-xtrabackup_lts = percona-xtrabackup_8_4;
+  # temporarily, latest LTS and Innovation release are equal
+  percona-xtrabackup_innovation = percona-xtrabackup_8_4;
```

Comment:
I'd vote for this to be in `aliases.nix`.

Reply:
Thanks for catching this, I did not see any announcements in just the release notes.